### DISCUSSION TITLE
tutorials/editor/using_the_android_editor

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#199] [User-contributed notes]
# tutorials/editor/using_the_android_editor

In 2023, we added an Android port of the editor that can be used to work on new or existing projects on Android devices. Android devices support: The Android editor requires devices running Android...

https://docs.godotengine.org/en/stable/tutorials/editor/using_the_android_editor.html

**Comment by llama-nl** [comment]
It is possible to use Android Plugins inside the Android Editor?

**Reply by Calinou** [reply]
As far as I know, this is not possible as there is no way to build them within the Android editor (Android plugins contain Java code).

**Reply by kaviagoct26-creator** [reply]
Not good

**Comment by JD-The-65th** [comment]
Another good keyboard to link would be [FUTO Keyboard.](https://keyboard.futo.org) It's source-available, they say it's in alpha but I've been daily driving it for months and have gotten ZERO issues, aside from ironically the link to pay them not working.

**Reply by maciek134** [reply]
Not open source at all - source available, given their custom license (people never learn...) prohibits commercial modification/distribution. But they don't claim it is, so please stop spreading misinformation.

**Reply by JD-The-65th** [reply]
> Not open source at all - source available, given their custom license (people never learn...) prohibits commercial modification/distribution. But they don't claim it is, so please stop spreading misinformation.

Oof, my bad. From the way I've heard people talk about it, I wasn't aware that it was Source Available. Not everyone checks every license on every piece of software they use.
